flagscale/serve/run_inference_engine.py

```diff
@@ -3,25 +3,33 @@
 import subprocess
 import sys
 
-from flagscale import serve
-
 # Compatible with both command-line execution and source code execution.
 try:
     import flag_scale
 except Exception as e:
     pass
 
+from flagscale import serve
+from flagscale.serve.utils import flatten_dict_to_args
+
 
 def vllm_serve(args):
     vllm_args = args["serve"]["model_args"]["vllm_model"]
 
     command = ["vllm", "serve"]
-    command.append(vllm_args["model-tag"])
-    for item in vllm_args:
-        if item not in {"model-tag", "action-args"}:
-            command.append(f"--{item}={vllm_args[item]}")
-    for arg in vllm_args["action-args"]:
-        command.append(f"--{arg}")
+    if vllm_args.get("model-tag"):
+        command.append(vllm_args["model-tag"])
```

Comment:
The naming convention, such as `model_tag`, should align with other modules and repositories familiar to users.

Reply:
model-tag --> model_tag, the same in:
"usage: vllm serve <**model_tag**> [options]"